01-node-tutorial/answers/04-names.js

```diff
@@ -0,0 +1,8 @@
+// local
+const secret = 'SUPER SECRET'
+
+// share
+const john = 'john'
+const peter = 'peter'
```

Comment:
Understanding question: what is the difference between shared and local? Why are the last two shared / where are they shared? How could we access secret in another file; what would we need to change?